woodwork/table_accessor.py

```diff
@@ -473,16 +473,19 @@ def wrapper(*args, **kwargs):
         # Directly return non-callable DataFrame attributes
         return dataframe_attr
 
-    def _get_subset_df_with_schema(self, cols_to_include):
+    def _get_subset_df_with_schema(self, cols_to_include, use_dataframe_order=True):
```

Comment:
I think we should add a specific test that verifies this parameter produces the desired ordering based on the `use_datafarme_order` value. 

It can be a simpler version of this test, focusing on just verifying the column ordering: https://github.com/alteryx/woodwork/blob/f18395909c57061415242b8b65cabed99477bd5c/woodwork/tests/accessor/test_table_accessor.py#L982

Reply:
✅ bf3f23f